Package@swift-5.1.swift

```diff
@@ -0,0 +1,35 @@
+// swift-tools-version:5.0
```

Comment:
Should these versions match the file names?

Reply:
I wouldn't change them. We don't need any newer feature in the manifests for the previous Swift version; therefor, I would just keep them as is.